packages/server/src/integrations/microsoftSqlServer.ts

```diff
@@ -186,10 +186,11 @@ class SqlServerIntegration extends Sql implements DatasourcePlus {
     try {
       const clientCfg: MSSQLConfig & sqlServer.config = {
         ...this.config,
-        port: +this.config,
+        port: +this.config.port,
         options: {
           encrypt: this.config.encrypt,
           enableArithAbort: true,
+          trustServerCertificate: true,
```

Comment:
Should this always be true? Should it not depend of any config? Any downside of having it always on?

Reply:
I suppose we could make it that if encryption is enabled we always allow it? I think having it as an option is more of a pain than just allowing self signed certs, they are very common now days afterall.

Reply:
Sounds good, just doublechecking that having it always on does not cause some edge case issues